src/JuMPDict.jl

```diff
@@ -85,6 +85,8 @@ macro gendict(instancename,T,idxpairs,idxsets...)
         else
             maprhs = :($(typename)(map(f,d.innerArray),d.name,d.indexsets,d.indexexprs))
         end
+        _wraplhs = :(_wrapper(d::$(typename),inner)) # helper function that wraps array into JuMPArray of similar type
```

Comment:
Is there a better name we can use than ``_wrapper``? This is basically a constructor.